en/getting-started/dashboard/index.md

```diff
@@ -64,8 +64,8 @@ This page is used to post official announcements or events of Cocos Creator, inc
 
 ## Learn
 
-The **Learn** page provides access to Cocos Creator user manuals, API manuals, and other help documentation, as well as tutorials, sample projects, and more.
+The **Learn** page provides access to the Cocos Creator user manual, API manual, and other help documentation, as well as tutorials, sample projects, and more.
 
 ![learn](index/learn.png)
 
-Developers can also visit the official Cocos website for more tutorials and other information by clicking on the icons in the lower left corner of each website, which currently includes WeChat, [Weibo](https://weibo.com/cocos2dx), [Twitter](https://twitter.com/cocos2dx), [BiliBili]( https://space.bilibili.com/491120849), [YouTube](https://www.youtube.com/channel/UCAsPLdpiAQbFuYqiZvi0P5A) and [GitHub](https://) github.com/cocos-creator/engine).
+Developers can also visit the official Cocos website for more tutorials and other information by clicking on the icons in the lower left corner of each website, which currently includes [Twitter](https://twitter.com/cocos2dx), [LinkedIn](https://www.linkedin.com/company/cocos-technologies/), [YouTube](https://www.youtube.com/channel/UCAsPLdpiAQbFuYqiZvi0P5A) and [GitHub](https://github.com/cocos-creator/engine).
```

Comment:
```suggestion
Developers can also visit the official Cocos website for tutorials and other information by clicking on the icons in the lower left corner of each website, which currently includes [Twitter](https://twitter.com/cocos2dx), [LinkedIn](https://www.linkedin.com/company/cocos-technologies/), [YouTube](https://www.youtube.com/channel/UCAsPLdpiAQbFuYqiZvi0P5A) and [GitHub](https://github.com/cocos-creator/engine).
```

Reply:
honestly, this paragraph doesn't make sense to me. 

 "by clicking on the icons in the lower left corner of each website,"

what website? I looked at docs and cocos.com and I don't see anything in the lower left corner of either of these...

Reply:
@slackmoehrle , here:

![image](https://user-images.githubusercontent.com/19702899/115503741-833e4100-a2a9-11eb-9078-2f3601f59e48.png)

Reply:
ok, I see. So it's not a website...

Reply:
```suggestion
Developers can also visit the official Cocos website for tutorials and other information by clicking on the icons in the lower-left corner of Cocos Dashboard. Connect with the team on [Twitter](https://twitter.com/cocos2dx), [LinkedIn](https://www.linkedin.com/company/cocos-technologies/), [YouTube](https://www.youtube.com/channel/UCAsPLdpiAQbFuYqiZvi0P5A) and [GitHub](https://github.com/cocos-creator/engine).
```